code_schemes/operator.json

```diff
@@ -0,0 +1,91 @@
+{
+  "SchemeID": "Scheme-f86cff0b",
+  "Name": "Operator",
```

Comment:
I think prefer making it clear that these are Somali operators - even though it's a change from name since ReDSS.